how-to/how-to-add-telerik-ui-for-nativescript-package.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to Include the Telerik UI for NativeScript Package
+description: Tutorial on how to include the Telerik UI for NativeScript package to AppBuilder project
+type: how-to
+page_title: How to add the Telerik UI for NativeScript package to an AppBuilder project
+slug: include-nativescript-ui-package
+position:
+tags: appbuilder, nativescript
+teampulseid:
+ticketid:
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package management</td>
+  </tr>
+  <tr>
+    <td>Mobile Development type</td>
+    <td>Native (NativeScript)</td>
+  </tr>
+</table>
+
+## Description
+This article provides the general steps required to include the Telerik UI for NativeScript package in your project.
+
+## Solution
+#### Telerik UI for NativeScript (Free)
+1. Open your project in AppBuilder.
+1. In the **Project Navigator** (**Solution Explorer** in Visual Studio), right-click the second node from the top and select **Manage Packages**.
+1. Find the `nativescript-telerik-ui` package inside the **NPM** panel and click **Install**.
```

Comment:
Usually, with technical documentation the context should come first. This means that you should first specify that users need to be on the NPM tab, then that they should find the telerik-ui package, and last - that they should click Install. Also, I'm guessing that picking the right version should also be stated here.

Basically, what you're looking for is something like that:

1. On the **NPM** tab, search for `nativescript-telerik-ui` and select the package.
1. (Optional) Select a package version.
1. Click **Install**.

Afterwards, it's not a bad idea to mention what happens - is the package.json file updated is there any confirmation about what has just happened. Should people commit to version control? Referencing the plugin in your code so that you can use it in the code is also essential. Alternatively, consider pointing to the getting started resources for the package.